### DISCUSSION TITLE
Collaborating on spreadsheets

### DISCUSSION THREAD

**Discussion by lucy-schick** [NewGraphEnvironment/restoration_framework#18] [General]
Just following up regarding @LauraNWRM question from today's meeting, "how can we collaborate on spreadsheets using Github". Al suggested we use Google Sheets. The best way to go about this would be to:

1.  Create a google sheets and copy/paste the data from your origin spreadsheet over
2. Allow editing permission to the UBR by sharing the google sheets, [link to how to share](https://support.google.com/docs/answer/9331169?hl=en#6.1). 
3. Create an issue on Github and post the link to the spreadsheet there

**Comment by NewGraphEnvironment** [comment]
Thanks Lucy.  This repo ( `restoration_framework` ) is a repository for developing restoration frameworks in general so linking project/area specific spreadsheets will be better done elsewhere.  The first 2 points look great.  Going forward. 


3. post link to spreadsheet in email for now. The link should be stable so only 1 email should be necessary
4. Once the folks that need access to the project specific repository ( `restoration_wedzin_kwa_2024` ) have made accounts and are added as collaborators OR `restoration_wedzin_kwa_2024` is made public we can create a discussion in ( restoration_wedzin_kwa_2024 ) and post the link there
